src/incubator/f5-bigip-ctlr/templates/f5-bigip-ctlr-deploy.yaml

```diff
@@ -44,16 +45,33 @@ spec:
               secretKeyRef:
                 name: {{ .Values.bigip_login_secret }}
                 key: password
+{{ else }}
+        volumeMounts:
+        - name: bigip-creds
+          mountPath: "/tmp/creds"
```

Comment:
this dir should be parameterized, and the presence of the value should trigger the non env var (default) behavior.

Reply:
Note: This requested change will impact directory references in multiple places in both files. 

The values file also needs to be updated with a section / comment that communicates where a volume mount path would go and how to use it.

Reply:
The "code" is from the documentation: https://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/v1.9/_downloads/06c1b4ae306f336ab3246375b7d01cae/example-bigip-credentials-directory.yaml

```
...
            "--credentials-directory=/tmp/creds",
...
          volumeMounts:
          - name: bigip-creds
            mountPath: "/tmp/creds"
            readOnly: true
      volumes:
      - name: bigip-creds
        secret:
          secretName: bigip-credentials
```

I'm open to suggestions on how to handle the volume mount path.  My thinking was to use a fixed path since the volume only applies to the controller itself and does not get used by any other resources.  Adding configurable values is reasonable with defaults from the documentation.  I'm not sure whether it would be a valid approach to embed the credentials in the image file itself and omit the volume mount and/or mount a non-secret volume, etc...